apps/mobile/src/features/approver/hooks.ts

```diff
@@ -0,0 +1,78 @@
+import { useToastContext } from '@/components/toast/toast-context';
```

Comment:
On this file, these hooks are really specific only to stacks transaction operations, that happen to use the approver ux pattern.

How might we structure the files to make that relationship clear? If stacks message signing hooks go here too, we could end up with a generic do-all Approver codebase that goes against the design of it being a self-contained use-agnostic pattern that takes care of approving things only.